Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const durationInHours = tasks.map(x => x.duration / 60);
+  const filterHours = durationInHours.filter(x => x >= 2);
+  for (let i = 0; i < filterHours.length; i++) {
+    filterHours[i] = filterHours[i] * hourlyRate;
+  }
   console.log(tasks, hourlyRate);
+
+  const sumEarnings = filterHours.reduce((total, amount) => total + amount);
+  const roundSumEarnings = Math.round(sumEarnings * 100) / 100;
```

Comment:
This rounding inside the `computeEarnings` function causes a loss of precision, which make this function unsuitable for use in further calculations. You should round only at the point where the result needs to be displayed to the user, which in this case in, or just before the console.log statement of line 63. The `.toFixed(2)` method can be used to fix the number of decimals to 2. No need for the `Math.round()` calculation.